stories/mocks/votes.ts

```diff
@@ -76,3 +85,62 @@ export const voteWithCorrectVoteWithUserVote = {
   ...voteWithUserVote,
   correctVote: 0,
 };
+
+export function makeVoteWithHistory(
+  vote: VoteT,
+  voted = false,
+  correctness = false,
+  staking = false,
+  slashAmount = BigNumber.from(0)
+) {
+  return {
+    ...vote,
+    voteHistory: {
+      uniqueKey: vote.uniqueKey,
+      voted,
+      correctness,
+      staking,
+      slashAmount,
+    },
+  };
+}
+
+const mockVoteHistory = {
+  voted: false,
+  correctness: false,
+  staking: false,
+  slashAmount: BigNumber.from(0),
+};
+
+export type VoteHistoryMockArgsT = {
+  vote?: VoteT;
+  voted?: boolean;
+  correctness?: boolean;
+  staking?: boolean;
+  slashAmount?: BigNumber;
+  length?: number;
+};
+
+function makeMockVoteHistory(args?: VoteHistoryMockArgsT) {
+  return {
+    ...mockVoteHistory,
+    ...args,
+    uniqueKey: `${Math.random()}`,
+    voted: args?.voted ?? Math.random() > 0.5,
+    correctness: args?.correctness ?? Math.random() > 0.5,
+    staking: args?.staking ?? Math.random() > 0.5,
+    slashAmount:
+      args?.slashAmount ?? bigNumberFromFloatString(`${Math.random() > 0.5 ? "-" : ""}${Math.random() * 100}`),
+  };
```

Comment:
<img width="539" alt="image" src="https://user-images.githubusercontent.com/53186719/194023117-9ad4ec47-47ca-4b89-a78d-ff9d6e28cee3.png">

If not staking the score is always 0

Reply:
These numbers are just randomly generated for display in the storybook — this logic is not in the dapp